CHANGELOG.md

```diff
@@ -19,20 +19,39 @@ The rules for this file:
 ## [Unreleased]
 
 ### Authors
-- ianmkenney
 
 ### Added
-<!-- New added features -->
 
 ### Fixed
-<!-- Bug fixes -->
 
 ### Changed
-<!-- Changes in existing functionality -->
-
-### Deprecated
-<!-- Soon-to-be removed features -->
 
 ### Removed
-<!-- Removed features -->
 
+## [1.0.0] - 2023-10-17
+
+### Authors
+- ianmkenney
+
+### Fixed
+- package deployment workflow enforces correct options regarding tests
+- formatting fixes in AUTHORS.md (PR #12)
+
+## [0.1.0] - 2023-10-13
+
+The waterdynamics MDAKit was created by Ian Kenney in 2023 and is based upon
+the MDAnalysis.analysis.waterdynamics analysis module originally written by
+Alejandro Bernardin in 2015. Additional contributors to the original source
+are listed in the AUTHORS.md file.
```

Comment:
Let's start with acknowledging the original code
```suggestion
The original `MDAnalysis.analysis.waterdynamics` was written by Alejandro 
Bernardin in 2015 and had been part of MDAnalysis since release 0.11.0,
https://docs.mdanalysis.org/2.6.1/documentation_pages/analysis/waterdynamics.html .
Ian Kenney created the `waterdynamics` MDAKit in 2023, based on the original
code in MDAnalysis. Additional contributors to the original source code are 
listed in the AUTHORS.md file.
```